packages/back-end/src/util/scheduleRules.ts

```diff
@@ -0,0 +1,34 @@
+import { ScheduleRule } from "../../types/feature";
+
+export const getCurrentEnabledState = (
+  scheduledRules: ScheduleRule[],
+  now: Date
+) => {
+  const sorted = scheduledRules
+    // Remove nulls
+    .filter((s) => s.timestamp)
+    // Convert timestamps to date objects
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .map((s: any) => ({
```

Comment:
Any insight on a better way to handle this? Typescript kept throwing an error on the line below since s.timestamp could be a `string | null`. Though, that will never happen since we're filtering nulls out just above this. 

I tried to create a new type, but unfortunately that didn't work.

Reply:
This is one way to force it, I'm sure there are others ... Should be ok as long as we are sure it's a string

```
    .map((s) => ({
      date: new Date(s.timestamp as string),
```

Reply:
Thank you!